src/settings_table.cpp

```diff
@@ -572,9 +572,11 @@ static void UpdateClientConfigValues()
 {
 	NetworkServerUpdateGameInfo();
 
+	InvalidateWindowData(WC_CLIENT_LIST, 0);
```

Comment:
`InvalidateWindowData` also marks the window dirty, so the `SetWindowClassesDirty` can be removed all together.

Reply:
I updated